openshift_cli_installer/libs/clusters/ipi_cluster.py

```diff
@@ -111,6 +116,15 @@ def _create_install_config_file(self):
         with open(os.path.join(self.cluster_info["cluster-dir"], "install-config.yaml"), "w") as fd:
             fd.write(yaml.dump(cluster_install_config))
 
+    def _set_docker_config_file(self):
+        ipi_docker_config_dir = os.path.dirname(self.docker_config_file)
+        if not ipi_docker_config_dir:
+            ipi_docker_config_dir = os.getcwd()
+
+        os.environ["DOCKER_CONFIG"] = ipi_docker_config_dir
```

Comment:
```suggestion
        os.environ["DOCKER_CONFIG"] = os.path.join(ipi_docker_config_dir, "config.json")
```

No need to copy it.

Also why sending only the file without a path is a valid usecase? 
in the help we have `Path to Docker config.json file.`, we expect path.

the code should be like:
`os.environ["DOCKER_CONFIG"] = self.docker_config_file`

Reply:
@myakove DOCKER_CONFIG needs to be a directory otherwise getting this error: 
```
error: unable to load --registry-config: reading JSON file "/home/cchetna/openshift-installer-files/docker-config.json/config.json": open /home/cchetna/openshift-installer-files/docker-config.json/config.json: not a directory
```
Also, this PR is needed because when tried running in openshift-cli-installer container (in jenkins), was getting error because there is no `~/.docker/config.json` (which is default path for registry credentials)
There are two ways I see to resolve this, either saving `self.docker_config_file` as `$DOCKER_CONFIG/config.json`, or moving it to `~/.docker/config.json`.